recipes/sigopt/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "sigopt" %}
+{% set version = "2.13.2" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "fe5d21c6586415618ae308ac6e1d8b0f7e68c3fb3b60782588d4e7eb900c736c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - requests
+  run:
+    - python
+
+test:
+  requires:
+    - requests
+  imports:
+    - sigopt
+    - sigopt.vendored
+
+about:
+  home: http://github.com/sigopt/sigopt-python
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'SigOpt Python API Client'
+
+extra:
+  recipe-maintainers:
+    - pfhayes
+    - alexandraj777 
```

Comment:
Are you ok being a maintainer on this, @alexandraj777?

Reply:
Yup, thanks for checking